requirements.txt

```diff
@@ -1,2 +1,3 @@
+ansible==2.5.9
```

Comment:
Missed this, please remove this bit, the Ansible version in the venv is controlled in the Makefile.